Autopilot/Test/Src/Test_Imu_skeleton.cpp

```diff
@@ -3,6 +3,11 @@
 #include "i2c.h"
 #include "mpu9255.h"
 
+#include "IMU_Mock.hpp"
+#include "IMU.hpp"
+
```

Comment:
Doesent #include <gtest/gtest.h> need to be here ? I guess not since this compiles lol, but seems weird, pretty sure the ::testing namespace comes from those headers anyway. This may be some Cmake magic that's solving the dependencies, in which case it would be best to include gtest, so things dont break if the build system changes at all in the future.

Reply:
#include <gtest/gtest.h> is the first line of the file